mofiebiger/ember-trivia
# ✨Adding global header and footer components + updated configurations/dependancies

# ✨ What Changed & Why

This PR is the first steps to creating a solid UI in the application. Two components were constructed `ui-header` & `ui-footer`. They take in both text input and images, so the components themselves can be dynamic by nature. The hope is to use these two components in the future within a modal. 

Additionally, a few dependancies were added. The first is [Ember truth helpers](https://www.npmjs.com/package/ember-truth-helpers) to logic within templates and [Ember FontAwesome](https://github.com/FortAwesome/ember-fontawesome) to display `<FaIcon />` components. Side Note: make surer to use `yarn add @fortawesome/free-solid-svg-icons` when installing (do not include `--dev`). 

# 👹 Before

<img width="1680" alt="Screenshot 2020-05-11 at 15 54 07" src="https://user-images.githubusercontent.com/43208685/81961658-91e89680-960a-11ea-87fe-5f490b64a239.png">

# 😇 After

<img width="1680" alt="Screenshot 2020-05-14 at 17 41 54" src="https://user-images.githubusercontent.com/43208685/81961684-9ca32b80-960a-11ea-984b-5e5c686392cc.png">

# 🌈 Testing

1. Visit [Ember-Trivia-Website](https://deploy-preview-3--kind-haibt-4424bd.netlify.app/)
1. Notice header & footer
1. Observe global style changes
